blog/siglens-1025x-faster-than-elasticsearch.html

```diff
@@ -0,0 +1,1075 @@
+                                We will use 6 shards so that we get better write and read performance. We will also disable replication to increase performance. Additionally let’s setup the timestamp field.
+                                <pre class="language-javascript"><code class="language-javascript">curl -k -u "elastic:" \
+    --location \
+    --request PUT 'https://<your_elasticsearch_host>:9200/_template/temp1' \
+    --header 'Content-Type: application/json' \
+    --data-raw '{
+        "index_patterns": "trips",
+        "settings": {
+            "number_of_shards": 6,
+            "number_of_replicas": 0
+        },
+        "mappings": {
+            "_source": {
+                "enabled": true
+            },
+            "properties": {
+                "timestamp": {
+                    "type": "date",
+                    "format": "epoch_millis"
+                }
+            }
+        }
+    }'</code></pre>
+                            </div>
+                        </div>
+                        <div>
+                            <h2 class="sub-heading mt-2 section" id="section4-4">Save the password for later
+                            </h2>
+                            If you don't know the password for the elastic user, reset it with
+                            <pre
+                                class="language-javascript"><code class="language-javascript">bin/elasticsearch-reset-password -u elastic</code></pre>The new password will be printed. Save it for later.
+                        </div>
+                        <div>
+                            <h2 class="sub-heading mt-2 section" id="section4-5">Run Elasticsearch
+                            </h2>
+                            <pre
+                                class="language-javascript"><code class="language-javascript">./bin/elasticsearch > elastic.log 2>&1</code></pre>
+                        </div>
+                        <div>
+                            <h2 class="sub-heading mt-2 section" id="section4-6">Ingest the Data</h2>In Terminal 2, clone siglens to use an ingester script from it.
+                            <pre
+                                class="language-javascript"><code class="language-javascript">cd /mnt nvme1n1
+git clone https://github.com/siglens/siglens.git
+cd siglens/tools/nyc-taxi-benchmark</code></pre>
+                                    In ingester.py, make these changes:
+                            <ul>
+                                <li>Change <span class="courier-font">index_data = '{"index": {"_index": "trips", "_type": "_doc"}}' to index_data = '{"index": {"_index": "trips"}}'</span></li>
+                                <li>In the curl section, add the <span class="courier-font">"-k"</span>, option, which allows faking SSL checks</li>
+                                <li>In the curl section, add <span class="courier-font">"-u", "elastic:<your-elastic-password>"</span></li>
```

Comment:
This might already be fixed, but I'm looking at the blog on https://sigscalr-blog.netlify.app/blog/siglens-1025x-faster-than-elasticsearch and `<your-elastic-password>` isn't showing up (so it just looks like `"elastic:"`. Is this already fixed or is the html trying to interpret that as a tag so it's not showing properly?

Same comment for everywhere else with `<your-elastic-password>`

Reply:
Use `&lt;` for `<` and `&gt;` for `>`.

`&lt;your-elastic-password&gt;` This way, it will be displayed as `<your-elastic-password>` without being interpreted as an HTML tag.

Reply:
Thanks @sonamgupta21
@AndrewHess it has been updated.